bin/plugin/commands/get-plugin-dir.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * External dependencies
+ */
+const path = require( 'path' );
+
+/**
+ * Internal dependencies
+ */
+const { log } = require( '../lib/logger' );
+
+exports.options = [
+	{
+		argname: '-s, --slug <slug>',
+		description: 'Slug to search out whether it is a plugin or module.',
+	},
+];
+
+/**
+ * Command to get directory for plugin/module based on the slug.
+ *
+ * @param {Object} opt Command options.
+ */
+exports.handler = async ( opt ) => {
+	doRunGetPluginDir( {
+		pluginsJsonFile: 'plugins.json', // Path to plugins.json file.
+		slug: opt.slug, // Plugin/module slug.
+	} );
+};
+
+/**
+ * Prints directory root for plugin or module based on the slug.
+ *
+ * @param {Object} settings Plugin settings.
```

Comment:
```suggestion

@param {Object} settings                 Plugin settings.
@param {string} settings.pluginsJsonFile Path to plugins JSON file.
@param {string} settings.slug            Slug for the plugin or module.
```

Same could be done in `get-plugin-version.js`